pkgs/desktops/deepin/library/gio-qt/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, cmake
+, pkg-config
+, wrapQtAppsHook
+, glibmm
+, doxygen
+, buildDocs ? false
```

Comment:
Whats the reason why this is off by default?

Reply:
fix in  https://github.com/NixOS/nixpkgs/pull/212979